packages/astro/tsconfig.json

```diff
@@ -6,6 +6,8 @@
     "declarationDir": "./dist",
     "module": "ES2022",
     "outDir": "./dist",
-    "target": "ES2021"
+    "target": "ES2021",
+    "jsx": "preserve",
+    "types": ["@types/dom-view-transitions", "network-information-types"]
```

Comment:
Using `types` make type acquisition explicit, which could affect other things (ex: we use `@types/node`), something to be on the lookout for

Reply:
What's the alternative?

Reply:
It depends how you need the types: 
- If you need them globally: Importing them in a `.d.ts` (like how our `env.d.ts` works)
- If you only need them in specific files: Importing the types in said files

Reply:
I'll note that it's not a problem, making type acquisition explicit, it's even good for performance. It's just something to be aware of

Reply:
Changed this to an import.